auto-instrument-handler/aws/aws-user-function.js

```diff
@@ -0,0 +1,157 @@
+/**
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * This code was copied from:
+ * https://github.com/aws/aws-lambda-nodejs-runtime-interface-client/blob/main/src/utils/UserFunction.ts
+ *
+ * This module defines the functions for loading the user's code as specified
+ * in a handler string.
+ */
+
+"use strict";
+
+const {
+  HandlerNotFound,
+  MalformedHandlerName,
+  ImportModuleError,
+  UserCodeSyntaxError
+} = require("./Errors.js");
+const path = require("path");
+const fs = require("fs");
+const FUNCTION_EXPR = /^([^.]*)\.(.*)$/;
+const RELATIVE_PATH_SUBSTRING = "..";
+
+/**
+ * Break the full handler string into two pieces, the module root and the actual
+ * handler string.
+ * Given './somepath/something/module.nestedobj.handler' this returns
+ * ['./somepath/something', 'module.nestedobj.handler']
+ */
+function _moduleRootAndHandler(fullHandlerString) {
+  let handlerString = path.basename(fullHandlerString);
+  let moduleRoot = fullHandlerString.substring(
+    0,
+    fullHandlerString.indexOf(handlerString)
+  );
+  return [moduleRoot, handlerString];
+}
+
+/**
+ * Split the handler string into two pieces: the module name and the path to
+ * the handler function.
+ */
+function _splitHandlerString(handler) {
+  let match = handler.match(FUNCTION_EXPR);
+  if (!match || match.length != 3) {
+    throw new MalformedHandlerName("Bad handler");
```

Comment:
That not an informative message..

Reply:
That's not my code 😂 
This is the message that the node runtime throws in this case. I don't want that the message will be different with our instrumentation 🤔